actors/stage/src/main/java/com/ea/orbit/actors/Stage.java

```diff
@@ -77,6 +79,9 @@
     @Wired
     Container container;
 
+    @Inject
```

Comment:
We should avoid doing this as if you aren't running in a container or DI framework it won't work.